tests/Doctrine/DBAL/Migrations/Tests/Stub/Functional/MigrateNotTouchingTheSchema.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Doctrine\DBAL\Migrations\Tests\Stub\Functional;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+class MigrateNotTouchingTheSchema extends AbstractMigration
```

Comment:
The spy doesn't work because the Version class instantiate the class based on it's name. And I didn't found a way to instantiate the spy based on it's name.